package.json

```diff
@@ -6,6 +6,7 @@
     "build": "tsc",
     "build-watch": "tsc -w",
     "tap": "tap test/*.test.* -Rspec --timeout=180 --node-path ts-node --test-file-pattern '/\\.[tj]s$/'",
+    "t": "tap test/remediation.test.* -Rspec --timeout=180 --node-path ts-node --test-file-pattern '/\\.[tj]s$/'",
```

Comment:
this was just for testing, please remove

Reply:
Done ✅